text-show-instances.cabal

```diff
@@ -156,6 +159,7 @@ library
                      , text-show             >= 3.4    && < 4
                      , time                  >= 0.1    && < 1.12
                      , unordered-containers  >= 0.2    && < 0.3
+                     , uuid                  >= 1      && < 2
```

Comment:
Would it suffice to use `uuid-types` instead of `uuid` here? `uuid-types` has fewer dependencies, and I don't think you need the extra stuff that `uuid` adds just to define an instance for `UUID`.